superset-frontend/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -696,8 +696,8 @@ class SqlEditor extends React.PureComponent {
   render() {
     const createViewModalTitle =
       this.state.createAs === CtasEnum.VIEW
-        ? 'Create View As'
-        : 'Create Table As';
+        ? 'Create view as'
+        : 'Create table as';
```

Comment:
CREATE VIEW AS and CREATE TABLE AS are outliers in this situation. We want to mimic the SQL language in this case so we're displaying this text as all caps. In this instance the correct form should be 'CREATE VIEW AS' and 'CREATE TABLE AS'